istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -233,6 +233,38 @@ func setupEnvoyLogConfig(kubeClient kube.CLIClient, param, podName, podNamespace
 	return string(result), nil
 }
 
+func printStatus(c *cobra.Command, kubeClient kube.CLIClient, statsType, podName, podNamespace string) error {
+	var stats string
+	var err error
+	if statsType == "" || statsType == "server" {
+		stats, err = setupEnvoyServerStatsConfig(kubeClient, podName, podNamespace, outputFormat)
+		if err != nil {
+			return err
+		}
+	} else if statsType == "cluster" || statsType == "clusters" {
+		stats, err = setupEnvoyClusterStatsConfig(kubeClient, podName, podNamespace, outputFormat)
+		if err != nil {
+			return err
+		}
+	} else {
+		return fmt.Errorf("unknown stats type %s", statsType)
+	}
+
+	fmt.Printf("\nThe result of pod %s:\n\n", podName)
```

Comment:
leftover debugging?

Reply:
So sorry, I don't understand what you mean. Here is to set a separator for the output results of multiple pods.

Reply:
Ah, apologies for the vague question.

I wasn't sure if this (`fmt.Printf("\nThe result of pod %s:\n\n", podName)`) was leftover code used for debugging or if it was added on purpose. I noticed it wasn't within this logic block before according to the diff. But if it makes the output cleaner, then makes sense!

Reply:
Why in this same function, you use both `fmt.Printf` and `fmt.Fprint`? Can you just use `fmt.Fprint(c.OutOrStdout(),...)`?

Reply:
Modified, thx